sdk/provisioning/Azure.Provisioning/tests/Primitives/ProvisioningOutputTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Provisioning.Expressions;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.Tests.Primitives;
+
+public class ProvisioningOutputTests
```

Comment:
Is there some tests that validate the enum change?

Reply:
test added